cmake/FindSCIPY_OPENBLAS32.cmake

```diff
@@ -0,0 +1,40 @@
+###############################################################################################
+# This file provides macros to process scipy-openblas32 external libraries.
+###############################################################################################
+
+# Include this only once
+include_guard()
+
+macro(get_scipy_openblas_libname SCIPY_OPENBLAS_LIB_NAME)
+    set(LIB_SUFFIX ".so")
```

Comment:
Is this variable https://cmake.org/cmake/help/latest/variable/CMAKE_SHARED_LIBRARY_SUFFIX.html supposed to do this?